app/src/main/res/values/styles.xml

```diff
@@ -563,9 +563,24 @@
         <item name="tint">?attr/primary_color</item>
     </style>
 
+    <style name="DialogIconWithSecondaryTint" parent="MaterialAlertDialog.Material3.Title.Icon.CenterStacked">
```

Comment:
Can we do something like this:

```
<style name="DialogIconWithTint.Secondary">
    <item name="tint">?attr/secondary_color</item>
</style>
```

This automatically inherits the `DialogIconWithTint` style.

Reply:
done.